oak_attestation_verification/src/lib.rs

```diff
@@ -18,17 +18,24 @@
 
 extern crate alloc;
 
+macro_rules! include_proto {
```

Comment:
If this is copied from somewhere else, could you mention it in a comment, and explain why that was not used in the first place? Otherwise the next person who looks around this code will think of replacing it again.

Reply:
Done.